course3/homework-string/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+project(MyString)
+
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -Wall -std=c++14")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -g -std=c++14")
```

Comment:
Don't add '-g' to the default set of flags. CMake will add it by itself when you configure the project in debug mode.